tools/pd-heartbeat-bench/config/config.go

```diff
@@ -0,0 +1,172 @@
+package config
+
+import (
+	"github.com/BurntSushi/toml"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	flag "github.com/spf13/pflag"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const (
+	defaultStoreCount        = 50
+	defaultRegionCount       = 1000000
+	defaultKeyLength         = 56
+	defaultReplica           = 3
+	defaultLeaderUpdateRatio = 0.06
+	defaultEpochUpdateRatio  = 0.04
+	defaultSpaceUpdateRatio  = 0.15
+	defaultFlowUpdateRatio   = 0.35
+	defaultRound             = 0
+	defaultSample            = false
+
+	defaultLogFormat = "text"
+)
+
+// Config is the heartbeat-bench configuration.
+type Config struct {
+	flagSet    *flag.FlagSet
+	configFile string
+	PDAddr     string
+	StatusAddr string
+
+	Log      log.Config `toml:"log" json:"log"`
+	logger   *zap.Logger
+	logProps *log.ZapProperties
+
+	StoreCount        int     `toml:"store-count" json:"store-count"`
+	RegionCount       int     `toml:"region-count" json:"region-count"`
+	KeyLength         int     `toml:"key-length" json:"key-length"`
+	Replica           int     `toml:"replica" json:"replica"`
+	LeaderUpdateRatio float64 `toml:"leader-update-ratio" json:"leader-update-ratio"`
```

Comment:
I think we still need command line arguments.

Reply:
It can be set through `--config`